src/verso-manual/VersoManual.lean

```diff
@@ -265,6 +265,8 @@ def traverse (logError : String → IO Unit) (text : Part Manual) (config : Conf
   let remoteContent ← updateRemotes false config.remoteConfigFile (if config.verbose then IO.println else fun _ => pure ())
   let mut state : Manual.TraverseState := {
     licenseInfo := .ofList config.licenseInfo,
+    extraCss := .insertMany {} config.extraJs
```

Comment:
I believe here it should be `extraCss := .insertMany {} config.extraCss` instead of `extraJs`? I'm trying to use it but it seems `extraCss` does not work without this change.

Reply:
In any case, https://github.com/leanprover/verso/pull/660 should do a lot to prevent a recurrence.